tests/rpc.rs

```diff
@@ -155,7 +155,7 @@ mod endpoints {
 
         assert_eq!(
             &response.hash.to_string(),
-            "75CA0F856A4DA078FC4911580360E70CEFB2EBEE"
+            "88D4266FD4E6338D13B845FCF289579D209C897823B9217DA3E161936F031589"
```

Comment:
Do the values of these not matter at all?

Reply:
As I understand, these tests only check that parsing work correctly, and do not test the actual correctness of the hash.
Also, the hash is of the sent transaction and not of the response right?

Reply:
> Also, the hash is of the sent transaction and not of the response right?

That's right